src/fshtypes/Configuration.ts

```diff
@@ -100,10 +100,10 @@ export type Configuration = {
   // refers to the IG.definiton.template definitions; not the publisher template in ig.ini.
   templates?: ImplementationGuideDefinitionTemplate[];
 
-  // DEPRECATED: Use of the template property is no longer recommended.  Authors should manage the
-  // ig.ini file themselves. Support for the template property will be removed in a future release.
+  // NO LONGER SUPPORTED: Use of the template property is no longer supported.  Authors should manage the
```

Comment:
Nevermind.  I see that we keep it so that we can detect when it has been used and log a more helpful error.

Can you update the comment to indicate why we keep the property?  Also, I think the comments below (on lines 105-106) are confusing since this property no longer works at all.

Reply:
Updated in b2e8d06, but see my question about if we do really even want those helpful errors. I'm not convinced anymore.